actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -81,6 +81,17 @@ def path_parameters
         get_header(PARAMETERS_KEY) || set_header(PARAMETERS_KEY, {})
       end
 
+      # Returns true if parameters are parse-able. Otherwise returns false.
+      #
+      #   GET /posts/5?x=1&y=2            | request.params_readable? => true
+      #   GET /posts/5?x[y]=1&x[y][][w]=2 | request.params_readable? => false
+      #
+      def params_readable?
+        !!parameters
```

Comment:
We avoid to write synthetic code just to return the singleton `true` or `false` so we don't need the `!!` here.